src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  padding: 20px 8px;
+}
+
+.form {
+  position: relative;
+}
+
+.form--big {
+  margin-bottom: 20px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  outline: none;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 3px;
+}
+
+.search--big {
+  padding: 23px 25px 23px 62px;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search--big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search--small {
+  padding: 10px 12px 11px 33px;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.search--small::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
```

Comment:
Can you avoid this repeated attribute?

Reply:
Keep only those values that really change. Placeholder text inherits the same styles as the input element, including font size, font family, font weight, color, background color, border, padding, and margin.